pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -2,7 +2,7 @@
 # Do not edit!
 
 {
-  version = "0.104.3";
+  version = "0.106.0b2";
```

Comment:
I merged a version newer than this here already: https://github.com/NixOS/nixpkgs/pull/81164/files

Reply:
I am trying to bump to latest now which is a bit newer, but `parse-requirements.py` is failing.

But I'm missing these commits too. I had cherry-picked some but I guess I need to actually rebase on master...